pallets/swaps/src/lib.rs

```diff
@@ -190,24 +203,28 @@ pub mod pallet {
 								Ok((
 									SwapStatus {
 										swapped: new_swap_amount_in,
-										pending: T::Balance::zero(),
+										pending: T::BalanceOut::zero(),
```

Comment:
Shouldn't this be `BalanceIn`? In this match arm, the inverse swap (`OrderBook<Out, In>` instead of `<In, Out>`) is larger. So the swapped amount is `Out` but the pending should be `In`, right?

Reply:
It is seen from the swap perspective, where swapped is always "in" and pending is always "out".

Note that `new_swap_amount_in` comes from `convert_by_market()` which returns `BalanceIn`. I didn't specified the bounds there because it's automatically inferred by the parameters.